digiwf-engine/digiwf-engine-service/src/main/java/io/muenchendigital/digiwf/engine/incidents/IncidentNotifierHandler.java

```diff
@@ -73,8 +74,15 @@ public void sendInfoMail(final IncidentEntity incidentEntity) throws IOException
                 incidentEntity.getProcessInstanceId() +
                 "/runtime";
 
+        String body = "In der Anwendung ist ein Incident aufgetreten.";
+
+        if (!StringUtils.isBlank(incidentEntity.getProcessDefinitionId())){
+            val processName = incidentEntity.getProcessDefinitionId().substring(0, incidentEntity.getProcessDefinitionId().indexOf(":"));
```

Comment:
This is a special case... For short process keys and versions the format is <ke>:<version> for long keys it might be just an UUID. So your index of ":" results in -1  and the substring will fail with wrong index. Please write a test for this more generic case.

Reply:
@zambrovski why not loading the ProcessDefinition via RepositoryService instead?

Reply:
mach ich über RepositoryService (siehe letzter commit)